sdk/tests/conformance/textures/misc/00_test_list.txt

```diff
@@ -11,6 +11,7 @@ copy-tex-image-and-sub-image-2d.html
 gl-pixelstorei.html
 gl-teximage.html
 origin-clean-conformance.html
+origin-clean-conformance-offscreencanvas.html
```

Comment:
Could you please add --min-version 1.0.4 here? This has been done for a while, just to track in which conformance version new tests were added.